### ISSUE TITLE
Local server > graphql-engine internal error

### ISSUE THREAD

**Issue by steph-ben** [PrefectHQ/prefect#3866]
## Description

When running `prefect server start`, the `hasura_1` container keep restarting with an internal error :

```
graphql-engine: internal error: Itimer: read(timerfd) failed,
    (GHC version 8.10.1 for x86_64_unknown_linux),
    Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
```

When running flows, it causes failures.

Google point me out to https://gitlab.haskell.org/ghc/ghc/-/issues/18033 , but I have no clue otherwise ...

## Expected Behavior

I expect the stack to run smoothly.

## Reproduction

```shell
conda install -c conda-forge prefect
prefect backend server
prefect server start
```

Output from cli :
```shell 
hasura_1    | graphql-engine: internal error: Itimer: read(timerfd) failed
hasura_1    |     (GHC version 8.10.1 for x86_64_unknown_linux)
hasura_1    |     Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
tmp_hasura_1 exited with code 139
...
hasura_1    | graphql-engine: internal error: Itimer: read(timerfd) failed
hasura_1    |     (GHC version 8.10.1 for x86_64_unknown_linux)
hasura_1    |     Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
tmp_hasura_1 exited with code 139
...
```

Output from hasura_1 container:
```shell
(prefect) ➜  ~ docker logs -f tmp_hasura_1
{"type":"pg-client","timestamp":"2020-12-17T11:03:31.645+0000","level":"warn","detail":{"message":"postgres connection failed, retrying(0)."}}
{"type":"pg-client","timestamp":"2020-12-17T11:03:31.645+0000","level":"warn","detail":{"message":"postgres connection failed, retrying(1)."}}
{"type":"startup","timestamp":"2020-12-17T11:03:31.645+0000","level":"error","detail":{"kind":"db_migrate","info":{"internal":"could not connect to server: Connection refused\n\tIs the server running on host \"postgres\" (172.31.0.2) and accepting\n\tTCP/IP connections on port 5432?\n","path":"$","error":"connection error","code":"postgres-error"}}}
{"type":"pg-client","timestamp":"2020-12-17T11:03:34.093+0000","level":"warn","detail":{"message":"postgres connection failed, retrying(0)."}}
{"type":"pg-client","timestamp":"2020-12-17T11:03:34.093+0000","level":"warn","detail":{"message":"postgres connection failed, retrying(1)."}}
{"type":"startup","timestamp":"2020-12-17T11:03:34.093+0000","level":"error","detail":{"kind":"db_migrate","info":{"internal":"could not connect to server: Connection refused\n\tIs the server running on host \"postgres\" (172.31.0.2) and accepting\n\tTCP/IP connections on port 5432?\n","path":"$","error":"connection error","code":"postgres-error"}}}
{"type":"startup","timestamp":"2020-12-17T11:03:36.730+0000","level":"error","detail":{"kind":"db_migrate","info":{"internal":{"statement":"CREATE EXTENSION IF NOT EXISTS pgcrypto SCHEMA public","prepared":false,"error":{"exec_status":"FatalError","hint":null,"message":"duplicate key value violates unique constraint \"pg_extension_name_index\"","status_code":"23505","description":"Key (extname)=(pgcrypto) already exists."},"arguments":[]},"path":"$","error":"pgcrypto extension is required, but it could not be created; encountered unknown postgres error","code":"postgres-error"}}}
graphql-engine: internal error: Itimer: read(timerfd) failed
    (GHC version 8.10.1 for x86_64_unknown_linux)
    Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
graphql-engine: internal error: Itimer: read(timerfd) failed
    (GHC version 8.10.1 for x86_64_unknown_linux)
    Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
graphql-engine: internal error: Itimer: read(timerfd) failed
    (GHC version 8.10.1 for x86_64_unknown_linux)
    Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
graphql-engine: internal error: Itimer: read(timerfd) failed
    (GHC version 8.10.1 for x86_64_unknown_linux)
    Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
graphql-engine: internal error: Itimer: read(timerfd) failed
    (GHC version 8.10.1 for x86_64_unknown_linux)
    Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
graphql-engine: internal error: Itimer: read(timerfd) failed
    (GHC version 8.10.1 for x86_64_unknown_linux)
    Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
graphql-engine: internal error: Itimer: read(timerfd) failed
    (GHC version 8.10.1 for x86_64_unknown_linux)
    Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
graphql-engine: internal error: Itimer: read(timerfd) failed
    (GHC version 8.10.1 for x86_64_unknown_linux)
    Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
```

## Environment

* OS & Docker : Running on VirtualBox

```
(prefect) ➜  ~ cat /etc/system-release
CentOS Linux release 7.5.1804 (Core)

(prefect) ➜  ~ docker --version
Docker version 19.03.8, build afacb8b
```

* Prefect
```
(prefect) ➜  ~ prefect version
0.14.0
(prefect) ➜  ~ prefect diagnostics
{
  "config_overrides": {},
  "env_vars": [],
  "system_information": {
    "platform": "Linux-3.10.0-862.el7.x86_64-x86_64-with-glibc2.17",
    "prefect_backend": "server",
    "prefect_version": "0.14.0",
    "python_version": "3.9.1"
  }
}
```

* Hasura container image :
```
hasura/graphql-engine:v1.3.0@sha256:f0166c4f2d415435a74cf9e9fb148e0efff6ee33857ddb30dfcdc1c1e25be2ee
```

**Comment by madkinsz** [comment]
Looks like this is a Haskell bug specific to virtual servers on Dell machines. According to https://gitlab.haskell.org/ghc/ghc/-/merge_requests/3137 it appears to have been fixed and backported to GHC 8.10.2 -- however the version listed above for GHC is 8.10.1. We may have to investigate bumping the Hasura version to fix this as we are using their image not a custom one.

**Comment by steph-ben** [comment]
Thanks for the quick feedback.
Using your branch, it actually fixes this error !

For the record, I'm running VirtualBox on Windows 10 on ThinkPad laptop.

Reading your comment, I currently left this issue open.
